src/main/scripts/src/app/shared/perspectives/documents/post-it/document/post-it-document.component.ts

```diff
@@ -104,38 +66,16 @@ export class PostItDocumentComponent implements OnInit {
   }
 
   private setEventListener(): void {
-    // disable scrolling when navigating using keys
+    // disable scrolling when navigating using up and down keys
     this.content.nativeElement.addEventListener('keydown', (key: KeyboardEvent) => {
-      [37, 38, 39, 40].includes(key.keyCode) && key.preventDefault();
+      [38, 40].includes(key.keyCode) && key.preventDefault();
```

Comment:
Isn't it better to use keycode constants? I think they are defined somewhere. It is not obvious now which keys are used here. We should avoid comments in code and make it self-describing instead.